test/src/lib.rs

```diff
@@ -77,4 +98,160 @@ mod tests {
         }
         Ok(())
     }
+
+    #[test]
+    fn test_simple_overflow_page() -> anyhow::Result<()> {
+        env_logger::init();
```

Comment:
You need to either use `Once` or `lazy_static` here because you're only supposed to call `env_logger::init()` once.

Reply:
env_logger has try_init where it will fail if already setup, so using that for now.